geo/Cargo.toml

```diff
@@ -19,6 +19,7 @@ num-traits = "0.2"
 serde = { version = "1.0", optional = true, features = ["derive"] }
 rstar = { version = "0.8" }
 geographiclib-rs = { version = "0.2" }
+log = "0.4.11"
```

Comment:
I've been waffling on doing this for a while, but finally bit the bullet. 

FIXES https://github.com/georust/geo/issues/596 if you're on board.